code/code/misc/crit_combat.cc

```diff
@@ -2344,24 +2320,113 @@ buf=format("$n's %s tears into $N's stomach exposing intestines!") %
 	if (v->desc)
 	  v->desc->career.crit_eviscerate_suff++;
 
-	return ONEHIT_MESS_CRIT_S;
-      case 88:
-      case 89:
-	// abdominal wound
-	// You plunge your %s into $N's abdoman and tear out causing a shower of blood
+	// triple damage
+        *dam *= 3;
 
-      case 90:
-      case 91:
-      case 92:
-      case 93:
+	return ONEHIT_MESS_CRIT_S;
       case 94:
       case 95:
+	if (!v->hasPart(WEAR_BACK))
+	  return 0;
+	if ((obj = v->equipment[WEAR_BACK])) {
+	  v->sendTo(COLOR_OBJECTS, format("Your %s saves you from a gory wound!\n\r") %
+		    fname(obj->name));
+	// triple damage
+        *dam *= 3;
+
+	  for (i=1;i<5;i++)
+	    if (v->equipment[WEAR_BACK])
+	      v->damageItem(this,WEAR_BACK,wtype,weapon,*dam);
+	  *part_hit = WEAR_BACK;
+	  return ONEHIT_MESS_CRIT_S;
+	}
       case 96:
       case 97:
+	// Side wound, vict stunned 6 rounds. 
+	if (!v->hasPart(WEAR_BACK))
+	  return 0;
+	buf=format("You plunge your %s deep into $N's side, stunning $M!") %
+		limbStr;
```

Comment:
From 1959 - limbStr=(weapon ? fname(weapon->name) : getMyRace()->getBodyLimbPierce(this));

Pretty sure even though the existing code refers to it as limbStr throughout this file, it can be a weapon as well.

Reply:
Yep, let's fix it to read `weaponStr`, as it's clearly meant to be a weapon (even if some races/classes use natural weapons).